src/pages/StreetView.vue

```diff
@@ -88,7 +88,13 @@
                 prominent
                 icon="mdi-clock-fast"
             >
-                {{ $tc('StreetView.countdownAlert', timeCountdown) }}
+                {{ $tc('StreetView.countdownAlert', remainingTime) }}
+                <v-progress-linear
+                    :active="isVisibleCountdownAlert"
+                    color="primary"
+                    v-model="countdownPercentage"
```

Comment:
```suggestion
                    v-model="remainingTime"
                    :buffer-value="timeCountdown"
```

You can remove le computed countdownPercentage with the  `buffer-value` prop
 https://vuetifyjs.com/en/api/v-progress-linear/#props-buffer-value

Reply:
As far as what I've tested, the total value is always `100`, as the image attached below.
![image](https://user-images.githubusercontent.com/33322229/190036850-8a10b13d-05fa-405e-a428-d589e4be880a.png)